js_modules/dagster-ui/packages/ui-core/src/hooks/useThrottledMemo.tsx

```diff
@@ -36,7 +34,6 @@ export const useThrottledMemo = <T,>(
   delay: number,
 ): T => {
   const [state, setState] = useState<T>(factory);
```

Comment:
@bengotow This line makes sure we fire the factory immediately

Reply:
Ahh nice didn't see this hiding here 🙌